firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -930,15 +895,7 @@ void loop()
     check_rx_timeouts();
         
     // check pressures on master cylinder (pressure from pedal)
-    smc.checkPedal();
-
-    // read and parse incoming serial commands
-    if( Serial.available() > 0 ) 
-    {
-        incomingSerialByte = Serial.read();
-        processSerialByte();
-    }
-      
```

Comment:
It was decided that this was messy and confusing for users. If they want to debug using serial then they are free to add it, but otherwise we though it best to remove it. (@evanlivingston @petebrink and I. )